cmd/kured/main.go

```diff
@@ -228,8 +228,8 @@ type RebootBlocker interface {
 // PrometheusBlockingChecker contains info for connecting
 // to prometheus, and can give info about whether a reboot should be blocked
 type PrometheusBlockingChecker struct {
-	// URL to contact prometheus API for checking alerts
-	promURL string
+	// labels to query get  alerts
```

Comment:
The Url will be configured when instantiating the prometheusClient, so no need to have it here, I guess?
Otherwise, we could add the PrometheusClient to this struct and use it? just like the "KubernetesBlockingChecker  struct. 
I'm not sure what to do here, if I'm being honest.

Reply:
incorporated the promClient into PrometheusBlockingChecker